src/cli/node/json_rpc_client.rs

```diff
@@ -1826,6 +1828,37 @@ pub fn priority(addr: SocketAddr, json: bool) -> Result<(), failure::Error> {
     Ok(())
 }
 
+pub fn query_stakes(
+    addr: SocketAddr,
+    validator: Option<String>,
+    withdrawer: Option<String>,
+) -> Result<(), failure::Error> {
+    let mut stream = start_client(addr)?;
+    let params = if validator.is_some() && withdrawer.is_some() {
+        Some(QueryStakesArgument::Key((
+            validator.unwrap(),
+            withdrawer.unwrap(),
+        )))
+    } else if validator.is_some() {
+        Some(QueryStakesArgument::Validator(validator.unwrap()))
+    } else if withdrawer.is_some() {
+        Some(QueryStakesArgument::Withdrawer(withdrawer.unwrap()))
+    } else {
+        None
+    };
```

Comment:
Normally, this is our preferred way to go about this, to avoid any redundant unwrapping:
```suggestion
    let params = match (validator, withdrawer) {
        (Some(validator), Some(withdrawer)) => Some(QueryStakesArgument::Key((
            validator,
            withdrawer,
        ))),
        (Some(validator), _) => Some(QueryStakesArgument::Validator(validator))
        (_, Some(withdrawer)) => Some(QueryStakesArgument::Withdrawer(withdrawer)),
        None => None,
    };
```